registry.k8s.io/images/k8s-staging-cluster-api/images.yaml

```diff
@@ -398,6 +400,7 @@
     "sha256:699e47bc5f10c8ebad7132a96e26624f564328766d31b0bbdc154c8cbb2ff085": ["v1.1.6"]
     "sha256:f154f5a8244daeaed1588d149ebf8f790d419d3575e96522848ca6af6292569d": ["v1.10.0-beta.0"]
     "sha256:ca136f3178c94e57b402b9541061afa49c53ef429abdba1b1bd7d4c110293aa3": ["v1.10.0-beta.1"]
+    "sha256:83dade316cc8fc3981432289b5a76ec1dc7a1c33cc9bab3b775288bb92a12c23": ["v1.10.0-rc.0"]
```

Comment:
`83dade316cc8` matches the kubeadm-bootstrap-controller [staging repo for v1.10.0-rc.0](https://console.cloud.google.com/gcr/images/k8s-staging-cluster-api/global/kubeadm-bootstrap-controller@sha256:83dade316cc8fc3981432289b5a76ec1dc7a1c33cc9bab3b775288bb92a12c23/details?invt=AbuOLA)